web/src/components/search/SearchPlaceholder.tsx

```diff
@@ -30,7 +30,7 @@ const SearchPlaceholder: React.FC<WithStyles<typeof styles>> = ({ classes }) =>
   return (
     <Box className={classes.root}>
       <Box display={'inline'}>
-        <MqText disabled inline>
+        <MqText disabled inline aria-label={i18next.t('search.search_aria')} aria-required='true'>
           {' '}
```

Comment:
Why is there a space character here?

Reply:
Good question. I removed it along with a few other unnecessary spaces and standardized the font so the spacing is consistent without them.
![Screenshot 2023-08-04 at 08 47 33](https://github.com/MarquezProject/marquez/assets/68482867/12a63f58-d4b7-46ec-a679-41dca1efdf29)